src/applications/vaos/referral-appointments/components/ReferralList.jsx

```diff
@@ -3,45 +3,64 @@ import PropTypes from 'prop-types';
 import { useHistory } from 'react-router-dom';
 import PendingReferralCard from './PendingReferralCard';
 import { routeToNextReferralPage } from '../flow';
+import InfoAlert from '../../components/InfoAlert';
+import NewTabAnchor from '../../components/NewTabAnchor';
 
-const ReferralList = ({ referrals }) => {
+const ReferralList = ({ referrals, referralsError }) => {
   const history = useHistory();
   const handleReferralClick = (e, referralId) => {
     e.preventDefault();
     routeToNextReferralPage(history, 'referralsAndRequests', referralId);
   };
-  // return if referrals is empty
+  if (referralsError) {
+    return (
+      <InfoAlert
+        status="error"
+        headline="We’re sorry. We’ve run into a problem"
+      >
+        We’re sorry. We can’t retrieve your community care referrals at this
+        time. Please try again later.
+      </InfoAlert>
+    );
+  }
   if (referrals.length === 0) {
-    return null;
+    return (
+      <div
+        data-testid="no-referral-content"
+        className="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-y--3"
+      >
+        <h2 className="vads-u-margin--0 vads-u-margin-bottom--2p5 vads-u-font-size--md">
+          You don’t have any referrals
+        </h2>
+        <p className="vads-u-margin-bottom--0">
+          If you think you should have referrals here, call your{' '}
+          <NewTabAnchor href="/find-locations">VA health facility</NewTabAnchor>
+        </p>
+      </div>
+    );
   }
   return (
-    <>
-      <h2 data-testid="referrals-heading">Community care referrals</h2>
-      <p data-testid="referrals-text">
-        Your care team approved these community care referrals. You can schedule
-        appointments with these providers now.
-      </p>
-      <ul
-        className="vads-u-padding-left--0 vads-u-margin-top--0"
-        data-cy="requested-appointment-list"
-      >
-        {referrals.map((referral, index) => {
-          return (
-            <PendingReferralCard
-              key={index}
-              index={index}
-              referral={referral}
-              handleClick={handleReferralClick}
-            />
-          );
-        })}
-      </ul>
-    </>
+    <ul
+      className="vads-u-padding-left--0 vads-u-margin-top--0"
+      data-testid="referral-list"
+    >
+      {referrals.map((referral, index) => {
+        return (
+          <PendingReferralCard
+            key={index}
+            index={index}
+            referral={referral}
+            handleClick={handleReferralClick}
+          />
+        );
+      })}
+    </ul>
   );
 };
 
 ReferralList.propTypes = {
-  referrals: PropTypes.arrayOf(PropTypes.string).isRequired,
+  referrals: PropTypes.arrayOf(PropTypes.object).isRequired,
+  referralsError: PropTypes.bool.isRequired,
```

Comment:
You have this set to required, but have a check for it as a conditional prop on line 15. This is causing a console error when there is no error.

Reply:
Oh never mind, I see its a bool. You need to update `const [requestsError, setRequestsError] = useState(null);` to `const [requestsError, setRequestsError] = useState(false);` in your hook